api/classes/base_export_cache.js

```diff
@@ -59,7 +59,7 @@ class ExportCache extends BaseCache {
           throw new Error(`Token ${saltedToken} provided for ${this.resourceName} is invalid`);
         }
 
-        return next(null, cachedId);
+        return next(null, parseInt(cachedId));
```

Comment:
our linter complains about it. I looked this up and according to eslint, since we're using an ES6 compliant environment (node >6), the radix parameter is redundant. https://eslint.org/docs/rules/radix

Reply:
Ah, I wasn't aware the default behaviour changed, good to know!